plugins/signals-node/report.api.md

```diff
@@ -22,6 +22,10 @@ export class DefaultSignalsService implements SignalsService {
 // @public (undocumented)
 export type SignalPayload<TMessage extends JsonObject = JsonObject> = {
   recipients:
+    | {
+        type: 'users';
+        entityRefs: string[];
+      }
```

Comment:
I am a but unclear on how the old contract was problematic, I think. It did support two types of input, but didn't have any optionality. How could it break?